From c31693e992d3f37706d6aa9078fe27384a40e17e Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Wed, 9 Sep 2015 12:01:50 +0200
Subject: [PATCH 1/2] Exclude mounts when scanning recent locations

Mount points are excluded for all other cases by default, so exclude
them for recent locations also.

https://bugzilla.gnome.org/show_bug.cgi?id=663603
---
 src/baobab-location.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/baobab-location.vala b/src/baobab-location.vala
index 4c47b7a..0722ddf 100644
--- a/src/baobab-location.vala
+++ b/src/baobab-location.vala
@@ -142,7 +142,7 @@ namespace Baobab {
                 get_fs_usage ();
             }
 
-            scanner = new Scanner (file, ScanFlags.NONE);
+            scanner = new Scanner (file, ScanFlags.EXCLUDE_MOUNTS);
         }
 
         public Location.for_file (File file_, ScanFlags flags) {
-- 
2.4.3

